### DISCUSSION TITLE
Video download is failed

### DISCUSSION THREAD

**Discussion by Avichal14** [aclap-dev/video-downloadhelper#546] [Help and Q&A]
### Your question

There is blob video in how to download that?

### Addon & CoApp version and details

 [hls @ 0x14d7048a0] Error when loading first segment 'https://qcdn.spayee.in/spees/w/o/62d540c30cf29665aa092a8d/v/65f583a95e72ca16957423dc/u/65114a53e4b0d1c15b125a7e/t/86ecfb55a118ef4265d8935d28069bf6/p/assets/videos/62d540c30cf29665aa092a8d/2024/03/16/65f583a95e72ca16957423dc/hls_2M_000.ts'
[in#0 @ 0x14d7047a0] Error opening input: Invalid data found when processing input
Error opening input file https://qcdn.spayee.in/spees/w/o/62d540c30cf29665aa092a8d/v/65f583a95e72ca16957423dc/u/65114a53e4b0d1c15b125a7e/t/86ecfb55a118ef4265d8935d28069bf6/p/assets/videos/62d540c30cf29665aa092a8d/2024/03/16/65f583a95e72ca16957423dc/hls_2M_.m3u8.
Error opening input files: Invalid data found when processing input
this error showing every time

**Comment by GrampaWildWilly** [comment]
You need to supply a proper problem report.  Refer to #525.

**Comment by Mattmjs** [comment]
This one is identical to #513 in that it is unable to open a key file. I don't think there is a solution yet.
